index.html

```diff
@@ -0,0 +1,78 @@
+                    Already have an Intuit account? <br>
+                    <a href="#">I forgot my User ID or Password</a>
+                </div>
+                <div id="sign-in-button-container">
+                    <button class="submit-button" id="sign-in-button">
+                        <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/icon-whitelock@3x.png" alt="padlock" class="padlock-closed-image-input">
+                        Sign in
+                    </button>
+                </div>
+            </div>
+        </div>
+        <header>
+            <div id="header-section">
+                <span id="header-image"></span>
+            </div>
+        </header>
+        <main>
+            <div id="icons-container">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/mint-favicon.svg" alt="icon1" class="image-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/mint-text.svg" alt="icon1" class="image-icon" id="mint-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/quickbooks-favicon@2x.svg" alt="icon1" class="image-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/quick-books-text.svg" alt="icon1" class="image-icon" id="quick-books-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/tt-favicon@2x.svg" alt="icon1" class="image-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/turbo-tax-text.svg" alt="icon1" class="image-icon" id="turbo-tax-icon">
+            </div>
+            <div id="main-container">
+                <div id="main-form-container">
+                    <div id="form-description-container">
+                        <h2>Create an Intuit account</h2>
+                        <p>One account for everything Intuit, including Mint.<br><a href="#" target="_blank">Learn more</a></p>
+                    </div>
+                    <form>
+                        <fieldset>
+                            <label for="email-field">Email</label>
+                            <input type="email" id="email-field" name="email">
+                            <label for="phone-field">Phone (Recommended)</label>
+                            <input type="tel" id="phone-field" name="phone">
+                            <span id="phone-warning">Standard call, messaging or data rates may apply.</span>
+                            <label for="password-field">Password</label>
+                            <input type="password" id="password-field" name="password">
+                            <img src="https://accounts.intuit.com/IUS-Plugins/v2/images/lock-left-grey@3x.png" alt="lock" id="padlock-image-input">
+                        </fieldset>
+                        <button class="submit-button">
```

Comment:
Set type attribute of button to submit. (See instruction on pathwright)

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png)

Reply:
Hey there! Thanks for taking a look at our project, and thanks for your feedback. Regarding those alignment issues you see at the top of the page, and at the terms and privacy section, we couldn't reproduce that error at all. We tested our page on several different browsers, with different viewport sizes, and they're looking right. I'll share a couple of screens to show you.

![edge](https://user-images.githubusercontent.com/39179783/70089963-f6351200-15f7-11ea-9c24-066fe42fa937.PNG)

![chrome](https://user-images.githubusercontent.com/39179783/70089981-ff25e380-15f7-11ea-86a1-b9cde09a84fb.PNG)

We also tried with different fonts, thinking that maybe the one we chose wasn't loading correctly on your end, but the page was still displaying correctly.

We still adjusted the margins at the top and the terms section just in case, they should be displaying a bit lower in the page now.

Reply:
First of all thanks again for the review of our project, I just want to add pictures of the webpage in my side, I am in mac and the display of the page looks good in Chrome and Firefox. Any comment to clarify the about the display in your side would be greatly appreciated.

<img width="1680" alt="Screen Shot 2019-12-03 at 3 39 24 PM" src="https://user-images.githubusercontent.com/48039473/70090726-bca8da80-15e8-11ea-8d11-b47eb9a0d620.png">
<img width="1680" alt="Screen Shot 2019-12-03 at 3 37 16 PM" src="https://user-images.githubusercontent.com/48039473/70090735-c3375200-15e8-11ea-8f85-4397f871acfc.png">